static-assets/components/cstudio-contextual-nav/targeting.js

```diff
@@ -308,14 +321,17 @@ CStudioAuthoring.ContextualNav.TargetingMod = CStudioAuthoring.ContextualNav.Tar
                       break;
                     case "datetime":
                       var $controlEl = $(controlEl),
-                          $dateTimePicker = $controlEl.find('.date-picker');
+                          $dateTimePicker = $controlEl.find('.date-picker'),
+                          $timeZonePicker = $controlEl.siblings('.timezone-container').find('.zone-picker');
 
                       if (currentProp.default_value === '') {
                         $dateTimePicker.val(currentProp.default_value);
+                        $timeZonePicker.hide();
```

Comment:
why hidden here? what’s the logic?

Reply:
When datetime selector is cleared (input empty) it hides the timezone selector as it is not needed (you can save empty value on the datetime selector)

Reply:
Let's leave it visible - just like the timepicker is all the time.